src/main/kotlin/com/team4099/robot2021/commands/climber/MoveClimber.kt

```diff
@@ -9,6 +10,10 @@ class MoveClimber(val pos: Constants.ClimberPosition): CommandBase() {
     addRequirements(Climber)
   }
 
+  override fun initialize() {
+    Logger.addEvent("MoveClimber", "Climber Moved")
```

Comment:
should add the position - something like
```
"Climber moved to $pos"
```
will include the position in the output string